libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFOAuthSessionRefresherTests.m

```diff
@@ -49,28 +49,6 @@ - (void)tearDown {
     [super tearDown];
 }
 
-- (void)testSuccessfulRefresh {
```

Comment:
Why remove that test?

Reply:
@wmathurin That test is redundant. We have refreshtests in the RestAPI. This test will now need user to be created in the refresh flow otherwise.